docs/reference/docs/reindex.asciidoc

```diff
@@ -484,6 +484,18 @@ Use in conjunction with `max_docs` to control what documents are reindexed.
 Set to a list to reindex select fields. 
 Defaults to `true`. 
 
+`type`:::
```

Comment:
I think we should leave out this part of the change, since it solely concerns the source type and this is just a filter on the source. I think adding it here increases the risk of it being used, so would kind of prefer not to add this now.

It is also unrelated to the issue, since regardless of specifying this filter, the type from source is disregarded when indexing into dest.

Reply:
I'm okay with removing this. As you mentioned, this isn't the main problem.